src/main/java/com/minecolonies/coremod/colony/crafting/ItemStorageFactory.java

```diff
@@ -42,9 +52,12 @@
 
     @NotNull
     @Override
-    public ItemStorage getNewInstance(@NotNull final ItemStack stack, final int size)
+    public ItemStorage getNewInstance(@NotNull final ItemStack stack, final int size, final boolean ignoreDamage, final boolean ignoreNBT)
     {
-        return new ItemStorage(stack, size, false);
+        ItemStorage newItem = new ItemStorage(stack,false, false);
```

Comment:
Why does this not use the arguments?

Reply:
Another good catch. I'd blame the beverages, but there was none of interest.